### PR TITLE
JOSS paper edits/comments by Chris

### PR DESCRIPTION
**Pull request by ashjbarnes**
*Replicating Chris's PR to point to joss-paper*

Hi @ashjbarnes @navidcy 
Following @aekiss and making my comments through a PR on a separate branch. 

I've made a few mostly minor modifications to the paper.md file. I've included a referece to Mike Herzfeld's work developing the original implementation of the radiation boundary conditions in MOM4, which includes a number of rules-of-thumb regarding regional domain parameters (grid resolution, time step, etc..). I also agree with @aekiss that there was a lot of useful information in the commented out sections of the paper, that, if space isn't an issue, I think could be included. 

A few brief comments
1. Worth noting that MOM6, while improving some aspects of the regional modelling, has also lost some capabilities. Noteably, a number of radiative schemes are yet to be implemented on the C-grid. Not sure if we need this level of detail in the paper. 
2. The major advantage of using MOM6 in regional modelling is to access the advantages of the new ocean model (ALE, better tracer advection schemes enabling larger time-steps, etc...);
3. Do we need step-by-step guide to what inputs are required, and what the expected outputs are? Or is this best handled within the code itself?
4. How removed from the NCI/Gadi environment do we need to be to make this work open for a general audience? 

Great work so far. I don't think you are far from having something ready to submit.

### CODE REVIEW

**Comment by navidcy** [comment]
> 1. Worth noting that MOM6, while improving some aspects of the regional modelling, has also lost some capabilities. Noteably, a number of radiative schemes are yet to be implemented on the C-grid. Not sure if we need this level of detail in the paper.

True. But let's not start comparing MOM5 and MOM6 in this paper. It's beyond the scope. We should only mention what is relevant for the package.

> 2. The major advantage of using MOM6 in regional modelling is to access the advantages of the new ocean model (ALE, better tracer advection schemes enabling larger time-steps, etc...);

Good point!

> 3. Do we need step-by-step guide to what inputs are required, and what the expected outputs are? Or is this best handled within the code itself?

It can go either way. At the moment we mention briefly and point to the documented examples.

> 4. How removed from the NCI/Gadi environment do we need to be to make this work open for a general audience?

We need to be totally removed! A package cannot be only applicable to particular machine. Imagine the [Nature paper](https://www.nature.com/articles/s41586-020-2649-2) titled "Array programming with NumPy on Australia's Gadi HPC". I doubt it would have the same audience...

Why you ask @ChrisC28 -- did you have anything in mind? As it stands now the package is completely removed from NCI-specifics. Only one of the two examples is NCI-specific, the one that uses ACCESS-OM2 output as forcing. But our plan is to remove this example from the package's documentation and move it to the cosima-recipes (see https://github.com/COSIMA/cosima-recipes/issues/326) and then add a link to the cosima-recipes in this package's docs.

**Comment by ashjbarnes** [comment]
Thanks Chris! I suggest we have a discussion here and resolve Chris's dot points one by one.

Regarding the commented out sections, given the feedback of @aekiss and @ChrisC28, I'll put them back in in this PR and then @navidcy perhaps we could then re-trim them with any information you think is excessive? 

1. I tend to agree that these details might be beyond what we need to include
2. This is a good point though and helps to contextualise why it's worth investing in MOM6 tools|
3. Inputs should be clear from the README of the package itself, but I hadn't thought about the outputs. I think the output files themselves ought to be expanded on in the documentation, if not the paper. Do you think it's sufficient that we say the package generates everything that's needed to run MOM6 except the executable itself without listing and explaining all of the components?
4. At this stage it's pretty devoid of NCI specific stuff (except for the access om2 example which we're still trying to migrate to cosima examples). One payu-centric thing is still present ( #93 )

**Comment by navidcy** [comment]
Let's merge this and then bring back the commented txt after we first ensure that appropriate modifications occur so that the text flows?

File: paper/paper.bib [hunk 1]
```diff
@@ -155,4 +155,17 @@ @article{griffies2012elements
...
+
+@article{Herzfeld2011,
+title = {Realistic test cases for limited area ocean modelling},
+journal = {Ocean Modelling},
+volume = {37},
+number = {1},
+pages = {1-34},
+year = {2011},
+issn = {1463-5003},
+doi = {https://doi.org/10.1016/j.ocemod.2010.12.008},
```
Comment (navidcy) [inline paper/paper.bib:168]:
```suggestion
doi = {10.1016/j.ocemod.2010.12.008},
```

File: paper/paper.md [hunk 2]
```diff
@@ -44,15 +44,13 @@ bibliography: paper.bib
 
 # Summary
 
-The Modular Ocean Model version 6 (MOM6) is a widely-used open-source general circulation ocean-sea ice model developed mainly at the Geophysical Fluid Dynamics Laboratory (GFDL) [@Adcroft2019MOM6].
-Among other improvements over its predecessor MOM5 [@griffies2012elements], MOM6 has more capable support for open boundary conditions and thus is becoming popular for regional ocean modeling studies (see, e.g., @gmd-16-6943-2023, @egusphere-2024-394) in addition to global configurations.
+The Modular Ocean Model version 6 (MOM6) is a relatively new yet widely-used open-source general circulation ocean-sea ice model developed mainly at the NOAA Geophysical Fluid Dynamics Laboratory (GFDL) [@Adcroft2019MOM6].
```
Comment (navidcy) [inline paper/paper.md:47]: it's not "relatively new"... it's been out for >5 years
let's not even debate what "new" is, the comment is a bit subjective so I suggest:

```suggestion
The Modular Ocean Model version 6 (MOM6) is a widely-used open-source general circulation ocean-sea ice model developed mainly at the NOAA Geophysical Fluid Dynamics Laboratory (GFDL) [@Adcroft2019MOM6].
```

**Comment by ashjbarnes** [comment]
> Let's merge this and then bring back the commented txt after we first ensure that appropriate modifications occur so that the text flows?

Sure sounds good! I'm happy with Chris's changes but can't approve now that it's my PR of course. I'll take your lead on bringing back the text since I'm sure I'll make a big git mess...

File: paper/paper.md [hunk 3]
```diff
@@ -44,15 +44,13 @@ bibliography: paper.bib
...
+MOM6 contains several improvements over its predecessor MOM5 [@griffies2012elements], including the implementation of the Abitrary-Lagrangian-Eulerian (ALE) vertical coordinates, more efficient tracer advection schemes, and state-of-the art parameterisations of sub-grid scale physics. MOM6 provides support for open boundary conditions and thus is becoming popular for regional ocean modeling studies (see, e.g., @gmd-16-6943-2023, @egusphere-2024-394) in addition to global configurations.
```
Comment (navidcy) [inline paper/paper.md:48]:
```suggestion
MOM6 contains several improvements over its predecessor MOM5 [@griffies2012elements], including the implementation of the Abitrary-Lagrangian-Eulerian (ALE) vertical coordinates [cite  https://doi.org/10.1029/2019MS001954], more efficient tracer advection schemes, and state-of-the art parameterisations of sub-grid scale physics.
 MOM6 provides support for open boundary conditions and thus is becoming popular for regional ocean modeling studies (see, e.g., @gmd-16-6943-2023, @egusphere-2024-394) in addition to global configurations.
```

File: paper/paper.md [hunk 4]
```diff
@@ -44,15 +44,13 @@ bibliography: paper.bib
...
-Thus, the package automates the re-gridding of all the required forcing input, takes care of all the metadata encoding, generates the regional grid, and deals with a few other necessary steps.
-This allows users to setup a regional MOM6 configurations using only Python and from a single Jupyter notebook.
+The input datasets can be on the Arakawa A, B, or C grids [@arakawa1977computational]; the package performs the appropriate interpolation using `xESMF` [@xesmf] under the hood, to put the everything on the C grid required by MOM6. Thus, the package automates the re-gridding of all the required forcing input, takes care of all the metadata encoding, generates the regional grid, and deals with a few other necessary steps. This allows users to setup a regional MOM6 configurations using only Python and from a single Jupyter notebook. The package allows the user to use MOM6's ALE vertical coordinates, regardless of the vertical coordinates of boundary forcing input. Rules-of-thumb to guide the user in setting grid parameters such as the regional domains resolution, can be found in @Herzfeld2011.
```
Comment (navidcy) [inline paper/paper.md:53]:
```suggestion
The input datasets can be on the Arakawa A, B, or C grids [@arakawa1977computational]; the package performs the appropriate interpolation using `xESMF` [@xesmf] under the hood, to put the everything on the C grid required by MOM6.
Thus, the package automates the re-gridding of all the required forcing input, takes care of all the metadata encoding, generates the regional grid, and deals with a few other necessary steps.
This allows users to setup a regional MOM6 configurations using only Python and from a single Jupyter notebook.
The package allows the user to use MOM6's ALE vertical coordinates, regardless of the vertical coordinates of boundary forcing input.
Rules-of-thumb to guide the user in setting grid parameters such as the regional domains resolution, can be found in the paper by @Herzfeld2011.
```

File: paper/paper.md [hunk 5]
```diff
@@ -76,7 +74,7 @@ However, the package's code design and modularity makes more complex configurati
 
 # Statement of need
 
-The learning curve for setting up a regional ocean model can be quite steep.
+The learning curve for setting up a regional ocean model can be quite steep, and it it not obvious for a new user what inputs are required, nor the appropriate format.
```
Comment (navidcy) [inline paper/paper.md:77]:
```suggestion
The learning curve for setting up a regional ocean model can be quite steep, and it is not obvious for a new user what inputs are required, nor the appropriate format of them.
```

**Review by navidcy — APPROVED** [review]
lgtm!
a few edits!
let's add citation to the ALE paper, https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2019MS001954

**Comment by navidcy** [comment]
@ashjbarnes I approved the PR but I also made some suggestions. Could you have a look and commit them if you agree or modify wherever you feel?

**Pull request merged**